src/main/java/teammates/ui/controller/Action.java

```diff
@@ -672,4 +676,22 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
     protected void excludeStudentDetailsFromResponseParams() {
         regkey = null;
     }
+
+    protected void verifyAccessibleForInstructorToFeedbackResponseComment(
```

Comment:
@chowyb Yes you are right.  Also is this inheritance is necessary as there is only one similar method and create a class for that is doable? 
In future there may be more use of it. So should i do it this (abstract class) way